src/lib/statistics/statistics_objects/equal_distinct_count_histogram.cpp

```diff
@@ -7,8 +7,8 @@
 #include <utility>
 #include <vector>
 
+#include <boost/sort/sort.hpp> // NOLINT
```

Comment:
If we add `// NOLINT`, please also add a description of why it was added.

In this case, I guess clang format is identifying `robin_map.h` as moves `sort.hpp` below it.